services/submission/src/test/java/app/coronawarn/server/services/submission/verification/TanTest.java

```diff
@@ -28,4 +29,19 @@ void validTanShouldReturnTanInstance() {
 
     assertThat(tan).isNotNull().hasToString(tanString);
   }
+
+  @Test 
+  void equalsTestTan(){
+    UUID tanUUID= UUID.randomUUID();
+    Tan tan = new Tan(tanUUID);
+    Assertions.assertTrue(tan.equals(tan));
+    Assertions.assertFalse(tan.equals(null);)
```

Comment:
It would be better to use the same style of assertions that all other tests use: eg. `assertThat(tan).isEqualTo(tan)`

Reply:
If I do this I will not test the method equals of the class Tan